### PR TITLE
Upgrade to 3.2.3

### PR DESCRIPTION
**Pull request by J0WI**
@DavidGoodwin http://postfixadmin.sourceforge.net/update-check.php announces 3.2 as latest stable version, is that correct?

### CODE REVIEW

File: library/postfixadmin [hunk 1]
```diff
@@ -2,17 +2,17 @@
 Maintainers: David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)
 GitRepo: https://github.com/postfixadmin/docker.git
 
-Tags: 3.2.2-apache, 3.2-apache, 3-apache, apache, 3.2.2, 3.2, 3, latest
+Tags: 3.2.3-apache, 3.2-apache, 3-apache, 3.2.3, 3.2, 3
```
Comment (yosifkit) [inline library/postfixadmin]: `apache` and `latest` tags removed on purpose? (and `fpm` and `fpm-alpine` removed below)

Comment (J0WI) [reply library/postfixadmin]: https://github.com/postfixadmin/docker/blob/master/generate-stackbrew-library.sh#L60

Comment (yosifkit) [reply library/postfixadmin]: It seems that check in generate should be loosened; as far as I can tell `update-check.php` only concerns itself with the major and minor of the version and ignores patch level entirely:

http://postfixadmin.sourceforge.net/update-check.php?version=3.2.0 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.2.1 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.2.3 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.2.z -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.3 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=4 -> `latest`

Comment (J0WI) [reply library/postfixadmin]: :ping_pong: @DavidGoodwin

**Comment by DavidGoodwin** [comment]
> @DavidGoodwin http://postfixadmin.sourceforge.net/update-check.php announces 3.2 as latest stable version, is that correct?

Hmm, probably. I'm not sure I remember how to go about updating that stuff ... I'll need to rummage in the SF docs.

**Comment by J0WI** [comment]
I've added the tags manually for now.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index b498426..740b502 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,10 +6,10 @@ postfixadmin:3.2
 postfixadmin:3.2-apache
 postfixadmin:3.2-fpm
 postfixadmin:3.2-fpm-alpine
-postfixadmin:3.2.2
-postfixadmin:3.2.2-apache
-postfixadmin:3.2.2-fpm
-postfixadmin:3.2.2-fpm-alpine
+postfixadmin:3.2.3
+postfixadmin:3.2.3-apache
+postfixadmin:3.2.3-fpm
+postfixadmin:3.2.3-fpm-alpine
 postfixadmin:apache
 postfixadmin:fpm
 postfixadmin:fpm-alpine
diff --git a/postfixadmin_fpm-alpine/Dockerfile b/postfixadmin_fpm-alpine/Dockerfile
index b1f2d6d..bf1d2af 100644
--- a/postfixadmin_fpm-alpine/Dockerfile
+++ b/postfixadmin_fpm-alpine/Dockerfile
@@ -13,7 +13,7 @@ RUN set -ex; \
 		sqlite-dev \
 		postgresql-dev \
 	; \
-	docker-php-ext-install \
+	docker-php-ext-install -j "$(nproc)" \
 		mysqli \
 		pdo_mysql \
 		pdo_pgsql \
@@ -29,8 +29,8 @@ RUN set -ex; \
 	apk add --virtual .postfixadmin-phpexts-rundeps $runDeps; \
 	apk del .build-deps
 
-ARG POSTFIXADMIN_VERSION=3.2.2
-ARG POSTFIXADMIN_SHA512=6c84cb215e69c52c26db0651e5d0d9d8bcb0a63b00d3c197f10fa1f0442a1fde44bb514fb476a1e68a21741d603febac67282961d01270e5969ee13d145121ee
+ARG POSTFIXADMIN_VERSION=3.2.3
+ARG POSTFIXADMIN_SHA512=d44addb9a3ca830caf55b603363054df561d659957f21cab7523465ebf02ca18abe7fcf298fe718d957d0b7bf5613e2dde69c78c26e0f7f6f595d79b28fe08ab
 
 ENV POSTFIXADMIN_VERSION $POSTFIXADMIN_VERSION
 ENV POSTFIXADMIN_SHA512 $POSTFIXADMIN_SHA512
diff --git a/postfixadmin_fpm/Dockerfile b/postfixadmin_fpm/Dockerfile
index 8a5bb7f..71e0a0e 100644
--- a/postfixadmin_fpm/Dockerfile
+++ b/postfixadmin_fpm/Dockerfile
@@ -12,7 +12,7 @@ RUN set -ex; \
 	libsqlite3-dev \
 	; \
 	\
-	docker-php-ext-install \
+	docker-php-ext-install -j "$(nproc)" \
 		mysqli \
 		pdo_mysql \
 		pdo_pgsql \
@@ -34,8 +34,8 @@ RUN set -ex; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	rm -rf /var/lib/apt/lists/*
 
-ARG POSTFIXADMIN_VERSION=3.2.2
-ARG POSTFIXADMIN_SHA512=6c84cb215e69c52c26db0651e5d0d9d8bcb0a63b00d3c197f10fa1f0442a1fde44bb514fb476a1e68a21741d603febac67282961d01270e5969ee13d145121ee
+ARG POSTFIXADMIN_VERSION=3.2.3
+ARG POSTFIXADMIN_SHA512=d44addb9a3ca830caf55b603363054df561d659957f21cab7523465ebf02ca18abe7fcf298fe718d957d0b7bf5613e2dde69c78c26e0f7f6f595d79b28fe08ab
 
 ENV POSTFIXADMIN_VERSION $POSTFIXADMIN_VERSION
 ENV POSTFIXADMIN_SHA512 $POSTFIXADMIN_SHA512
diff --git a/postfixadmin_latest/Dockerfile b/postfixadmin_latest/Dockerfile
index 6b18dd2..925886c 100644
--- a/postfixadmin_latest/Dockerfile
+++ b/postfixadmin_latest/Dockerfile
@@ -12,7 +12,7 @@ RUN set -ex; \
 	libsqlite3-dev \
 	; \
 	\
-	docker-php-ext-install \
+	docker-php-ext-install -j "$(nproc)" \
 		mysqli \
 		pdo_mysql \
 		pdo_pgsql \
@@ -34,8 +34,8 @@ RUN set -ex; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	rm -rf /var/lib/apt/lists/*
 
-ARG POSTFIXADMIN_VERSION=3.2.2
-ARG POSTFIXADMIN_SHA512=6c84cb215e69c52c26db0651e5d0d9d8bcb0a63b00d3c197f10fa1f0442a1fde44bb514fb476a1e68a21741d603febac67282961d01270e5969ee13d145121ee
+ARG POSTFIXADMIN_VERSION=3.2.3
+ARG POSTFIXADMIN_SHA512=d44addb9a3ca830caf55b603363054df561d659957f21cab7523465ebf02ca18abe7fcf298fe718d957d0b7bf5613e2dde69c78c26e0f7f6f595d79b28fe08ab
 
 ENV POSTFIXADMIN_VERSION $POSTFIXADMIN_VERSION
 ENV POSTFIXADMIN_SHA512 $POSTFIXADMIN_SHA512
```

</details>

**Comment by yosifkit** [comment]
Build test of #7121; cf56b2f4; `amd64` (`postfixadmin`):

```console

$ bashbrew build postfixadmin:3.2.3-apache
Building bashbrew/cache:bf50ea26ea8087b834b61723ea4ef88025108f871ca9c912bb4b4dd8ffe40510 (postfixadmin:3.2.3-apache)
Tagging postfixadmin:3.2.3-apache
Tagging postfixadmin:3.2-apache
Tagging postfixadmin:3-apache
Tagging postfixadmin:apache
Tagging postfixadmin:3.2.3
Tagging postfixadmin:3.2
Tagging postfixadmin:3
Tagging postfixadmin:latest

$ test/run.sh postfixadmin:3.2.3-apache
testing postfixadmin:3.2.3-apache
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build postfixadmin:3.2.3-fpm
Building bashbrew/cache:1aa6aa318a603bbe69da1d8978db5e1c62bb11471c0cde32f25c0d8448d42938 (postfixadmin:3.2.3-fpm)
Tagging postfixadmin:3.2.3-fpm
Tagging postfixadmin:3.2-fpm
Tagging postfixadmin:3-fpm
Tagging postfixadmin:fpm

$ test/run.sh postfixadmin:3.2.3-fpm
testing postfixadmin:3.2.3-fpm
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build postfixadmin:3.2.3-fpm-alpine
Building bashbrew/cache:46a35e60a8235b76cb22ea6349edd2ac7d7ef6212892beec6381289a27eda93e (postfixadmin:3.2.3-fpm-alpine)
Tagging postfixadmin:3.2.3-fpm-alpine
Tagging postfixadmin:3.2-fpm-alpine
Tagging postfixadmin:3-fpm-alpine
Tagging postfixadmin:fpm-alpine

$ test/run.sh postfixadmin:3.2.3-fpm-alpine
testing postfixadmin:3.2.3-fpm-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**